shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/JwtUtils.java

```diff
@@ -58,7 +58,7 @@ public static UserInfo getUserInfo() {
      */
     public static String getIssuer(final String token) {
         DecodedJWT jwt = JWT.decode(token);
-        return Optional.ofNullable(jwt).map(item -> item.getClaim("userName").asString()).orElse("");
+        return Optional.of(jwt).map(item -> item.getClaim("userName").asString()).orElse("");
```

Comment:
why modify Optional.ofNullable to Optional.of?   jwt won't it be Null？

Reply:
JWT#decode(String) implement is:
`
new JWTDecoder(parser, token)
`
it not null